src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -469,6 +469,18 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.show_command(
             'show', 'get', transform=lambda x: getattr(x, 'legal_hold', x))
 
+    with self.command_group('storage container-rm', command_type=blob_container_mgmt_sdk,
+                            custom_command_type=get_custom_sdk('blob', cf_blob_container_mgmt,
+                                                               resource_type=ResourceType.MGMT_STORAGE),
+                            resource_type=ResourceType.MGMT_STORAGE, min_api='2018-02-01') as g:
+        g.custom_command('create', 'create_container_rm')
+        g.command('delete', 'delete', confirmation=True)
+        g.custom_command('update', 'update_container_rm')
```

Comment:
typically we will recommend to use generic_update() for update command to expose additional --add, --set and --remove arguments for custom usage.